### ISSUE TITLE
pr_mask is not correct although the accuracy on CamVid test set is very high

### ISSUE THREAD

**Issue by KleinXin** [qubvel/segmentation_models.pytorch#13]
I used se_resnext50_32x4d encoder and Unet decoder to train the segmentation model by following the CamVid tutorial on the webpage

However, the pr_mask is not correct although the accuracy on CamVid test set is very high（iou - 0.7417）

valid: 100%|████████████████████████████████████████████| 233/233 [00:10<00:00, 21.86it/s, bce_dice_loss - 0.3587, iou - 0.7417, f-score - 0.8195]

![image](https://user-images.githubusercontent.com/13084622/57267910-61dfec80-70b4-11e9-8365-c090063406c3.png)

The trained model and codes can be found here

https://drive.google.com/drive/folders/0B6X3_r_lRbVUODRiM2RhMWMtNDk4NC00NmM1LWEyODEtMTdjNDI5MWJiMmQ4

**Comment by qubvel** [comment]
Hi, did you use tutorial without any changes?

**Comment by KleinXin** [comment]
Yes, I think so.


Can you get the model and script I shared with you？


| |
陈心
|
|
13681176447@163.com
|
签名由网易邮箱大师定制
On 5/7/2019 13:26，Pavel Yakubovskiy<notifications@github.com> wrote：

Hi, did you use tutorial without any changes?

**Comment by qubvel** [comment]
Yeah, I can. Did you try colab tutorial?
No any ideas whats wrong.

**Comment by crossknight** [comment]
I got the same problem with other dataset using se_resnext50_32x4d encoder too.

**Comment by qubvel** [comment]
Line 321 bug, change `model.predict` -> `best_model.predict`

**Comment by KleinXin** [comment]
@qubvel  problem solved, thank you very much

![Uploading image.png…]()
